forge/ee/lib/billing/index.js

```diff
@@ -211,6 +217,51 @@ module.exports.init = async function (app) {
                 }
             }
         },
+        updateTeamDeviceCount: async (team) => {
+            const billingIds = getBillingIdsForTeam(team)
+            if (!billingIds.device.product) {
+                return
+            }
+            const subscription = await app.db.models.Subscription.byTeam(team.id)
+            if (subscription) {
+                const deviceCount = await team.deviceCount()
+                const deviceFreeAllocation = team.TeamType.getProperty('deviceFreeAllocation') || 0
+                const billableCount = Math.max(0, deviceCount - deviceFreeAllocation)
```

Comment:
Question: I cannot see where `deviceFreeAllocation` is ever set (different PR? Future? From the License?)

Reply:
Also, should `deviceFreeAllocation` be covered in tests?

Reply:
`deviceFreeAllocation` will be defined as a property of the `TeamType` - (ref: https://github.com/flowforge/flowforge/issues/782#issuecomment-1241742266)

It defaults to 0 - as it stands we haven't decided what values we will apply (if any). The purpose of that part of this PR is to ensure we have this capability.


I will add some additional tests for it - just requires a bit of juggling around of things.